layouts/partials/ecosystem/registry/entry.html

```diff
@@ -1,95 +1,96 @@
-                "installTemplate" "ecosystem/registry/quickinstall/default.md"
-                "icon" "fa-brands fa-golang")
-            "go-collector" (dict
-                "urlPattern" "https://pkg.go.dev/%s"
-                "installTemplate" "ecosystem/registry/quickinstall/collector.md"
-                "icon" "fa-solid fa-box-open")
-            "nuget" (dict
-                "urlPattern" "https://www.nuget.org/packages/%s"
-                "installLine" "dotnet add package %s"
-                "installTemplate" "ecosystem/registry/quickinstall/default.md"
-                "icon" "fa-solid fa-box-open")
-            "pip" (dict
-                "urlPattern" "https://pypi.org/project/%s"
-                "installLine" "pip install %s"
-                "installTemplate" "ecosystem/registry/quickinstall/default.md"
-                "icon" "fa-brands fa-python")
-            "hex" (dict
-                "urlPattern" "https://hex.pm/packages/%s"
-                "installTemplate" "ecosystem/registry/quickinstall/hex.md"
-                "icon" "fa-brands fa-erlang")
-            "crates" (dict
-                "urlPattern" "https://crates.io/crates/%s"
-                "installLine" "cargo add %s"
-                "installTemplate" "ecosystem/registry/quickinstall/default.md"
-                "icon" "fab fa-rust")
-    -}}
+{{
+$remoteRegistries := dict
+"npm" (dict
+"urlPattern" "https://npmjs.com/package/%s"
+"installTemplate" "ecosystem/registry/quickinstall/default.md"
+"installLine" "npm install %s"
+"icon" "fab fa-npm")
+"packagist" (dict
+"urlPattern" "https://packagist.org/packages/%s"
+"installLine" "composer require %s"
+"installTemplate" "ecosystem/registry/quickinstall/default.md"
+"icon" "fa-solid fa-box-open")
+"gems" (dict
+"urlPattern" "https://rubygems.org/gems/%s"
+"installLine" "gem install %s"
+"installTemplate" "ecosystem/registry/quickinstall/default.md"
+"icon" "fa-solid fa-gem")
+"go" (dict
+"urlPattern" "https://pkg.go.dev/%s"
+"installLine" "go get %s"
+"installTemplate" "ecosystem/registry/quickinstall/default.md"
+"icon" "fa-brands fa-golang")
+"go-collector" (dict
+"urlPattern" "https://pkg.go.dev/%s"
+"installTemplate" "ecosystem/registry/quickinstall/collector.md"
+"icon" "fa-solid fa-box-open")
+"nuget" (dict
+"urlPattern" "https://www.nuget.org/packages/%s"
+"installLine" "dotnet add package %s"
+"installTemplate" "ecosystem/registry/quickinstall/default.md"
+"icon" "fa-solid fa-box-open")
+"pip" (dict
+"urlPattern" "https://pypi.org/project/%s"
+"installLine" "pip install %s"
+"installTemplate" "ecosystem/registry/quickinstall/default.md"
+"icon" "fa-brands fa-python")
+"hex" (dict
+"urlPattern" "https://hex.pm/packages/%s"
+"installTemplate" "ecosystem/registry/quickinstall/hex.md"
+"icon" "fa-brands fa-erlang")
+"crates" (dict
+"urlPattern" "https://crates.io/crates/%s"
+"installLine" "cargo add %s"
+"installTemplate" "ecosystem/registry/quickinstall/default.md"
+"icon" "fab fa-rust")
+-}}
```

Comment:
please revert this and other formatting changes

Reply:
ok